qutebrowser/mainwindow/tabbedbrowser.py

```diff
@@ -739,7 +739,14 @@ def on_load_finished(self, tab, ok):
             start = config.val.colors.tabs.indicator.start
             stop = config.val.colors.tabs.indicator.stop
             system = config.val.colors.tabs.indicator.system
-            color = utils.interpolate_color(start, stop, 100, system)
+            try:
+                url_str=self.widget.tab_url(idx).toString()
+                if url_str.find("qute://back") != -1 :
```

Comment:
I don't think there is any reason why that shouldn't be `url_str.startswith`.

Reply:
Let me elaborate:

- You only need `str.find` when you really want to know the position of a string in another string - which is very seldom, I don't think I've ever used it actually!
- Instead, the idiomatic way of doing this is `if "qute://back" in url_str:`
- However, as @toofar said, if you want whether it starts with it, do `url_str.startswith("qute://back")`
- However, I'd rather get the individual parts from the "intelligent" `QUrl` object instead of using string comparisons, which would mean `if (url.scheme(), url.host()) == ('qute', 'back'):`